templates/themes/app/forms/theme.html.twig

```diff
@@ -1,17 +1,21 @@
-{% extends 'bootstrap_4_layout.html.twig' %}
+{% extends 'bootstrap_5_layout.html.twig' %}
 
 {%- block form_row -%}
-    <div class="form-group {% if errors|length > 0 %}error-input{% endif %}">
-        {%- if 'checkbox' in block_prefixes or 'radio' in block_prefixes -%}
-            {{- form_widget(form) -}}
-            {{- form_label(form) -}}
-            {{- form_errors(form) -}}
-        {%- else -%}
-            {{- form_label(form) -}}
-            {{- form_widget(form) -}}
-            {{- form_errors(form) -}}
-        {%- endif -%}
-    </div>
+    {% if compound %}
+        {{- form_widget(form) -}}
+    {% else %}
+        <div class="form-group {% if errors|length > 0 %}error-input{% endif %}">
```

Comment:
I think we should have this wrapper by default, and kill it only for the hidden fields.